package.json

```diff
@@ -168,29 +178,30 @@
     "sinon": "^16.1.3",
     "stream-browserify": "^3.0.0",
     "string-replace-loader": "^3.1.0",
+    "terser-webpack-plugin": "^5.3.10",
     "tsx": "^4.7.0",
-    "typescript": "^5.3.3",
+    "typescript": "^5.4.5",
```

Comment:
Had to bump to avoid problems with TS being unable to correctly resolve linked deps in a monorepo.
https://github.com/microsoft/TypeScript/issues/42873#issuecomment-2065572017